### ISSUE TITLE
Front end for sign-up/sign-in pages

### ISSUE THREAD

**Issue by gautamgupta** [uwblueprint/help-lesotho#23]
Borrow design from existing screens (eg. search box, labels) - https://www.figma.com/file/Y1zyCnaqKtTMlZei53r8KJhs/HL-Prototype-1

 - [ ] Implement for sign-up page
 - [ ] Implement for sign-in page

**Comment by gautamgupta** [comment]
@carol-yao Do you think you can do a design for this too? :)

**Comment by carol-yao** [comment]
@GautamGupta yup for sure, what is supported for the login? Can I get a list of fields that we will have and will we be supporting any social media logins?
